content/influxdb/v2.1/notebooks/manage-notebooks.md

```diff
@@ -9,9 +9,45 @@ menu:
     parent: Notebooks
 ---
 
-View, update, and delete notebooks in the UI.
+Manage your notebooks in the UI:
+
+- {{% cloud-only %}}[Share a notebook](#share-a-notebook){{% /cloud-only %}}
+- {{% cloud-only %}}[Unshare a notebook](#unshare-a-notebook){{% /cloud-only %}}
+- [View or update a notebook](#view-or-update-notebooks)
+- [Delete a notebook](#delete-a-notebook)
+
+{{% cloud-only %}}
```

Comment:
@sanderson after adding the cloud-only tag, the indent format isn't applied (so numbering is off). 

![image](https://user-images.githubusercontent.com/47797004/143778653-4667e8e2-1a15-4cd0-a27d-4466f36c3c01.png)

Verified that removing the tag fixes the issue:
![image](https://user-images.githubusercontent.com/47797004/143778669-9466c773-9e51-478c-a16f-1d508aaf8795.png)

How best to handle?